stable/mongodb-replicaset/templates/mongodb-statefulset.yaml

```diff
@@ -185,7 +204,74 @@ spec:
               mountPath: /data/configdb
             - name: workdir
               mountPath: /work-dir
-    {{- with .Values.nodeSelector }}
+{{ if .Values.metrics.enabled }}
+        - name: metrics
+          image: "{{ .Values.metrics.repository }}:{{ .Values.metrics.tag }}"
+          imagePullPolicy: {{ .Values.metrics.imagePullPolicy | quote }}
+          command:
+            - sh
+            - -ec
+            - |-
+            {{- if .Values.auth.enabled }}
+              export MONGODB_URL=mongodb://$METRICS_USER:$METRICS_PASSWORD@localhost:{{ .Values.port }};
+            {{- else }}
+              export MONGODB_URL=mongodb://localhost:{{ .Values.port }};
+            {{- end }}
+              /bin/mongodb_exporter
+            {{- if .Values.tls.enabled }}
+              -mongodb.tls
+              -mongodb.tls-ca=/ca/tls.crt
+              -mongodb.tls-cert=/work-dir/mongo.pem
+            {{- end }}
+              -web.metrics-path={{ .Values.metrics.path }}
+              -web.listen-address=:{{ .Values.metrics.port }}
```

Comment:
There are timeout flags in the latest release I added in https://github.com/percona/mongodb_exporter/pull/100. Please make them configurable as well.